src/components/endpoints/SendTransaction.vue

```diff
@@ -25,229 +20,6 @@ const data = ref<string>('')
 const hasData = ref(false)
 const isPending = ref(false)
 
-const methods: MethodSelect[] = [
```

Comment:
This was just moved to a separate file to be reused in `SignTransaction.vue` component.